search-service/src/test/kotlin/com/egm/stellio/search/web/AnonymousUserHandlerTests.kt

```diff
@@ -0,0 +1,57 @@
+package com.egm.stellio.search.web
```

Comment:
why not put it directly in `EntityHandlerTest.kt` like it's done in `TemporalEntityOperationHandlerTest.kt` ?

Reply:
I also removed it from `TemporalEntityOperationHandlerTest.kt`. kind of dumb to duplicate the same test in each and every handler for a transversal topic.